src/cargo.rs

```diff
@@ -46,7 +46,7 @@ impl Rustflags {
     pub fn for_xargo(&self, home: &Home) -> String {
         let mut flags = self.flags.clone();
         flags.push("--sysroot".to_owned());
-        flags.push(home.display().to_string());
+        flags.push(format!("\"{}\"", home.display()));
```

Comment:
Isn't this `RUSTFLAGS`? The quotes won't do anything there. There currently just is no way to pass arguments with spaces via `RUSTFLAGS`. That is a cargo bug: https://github.com/rust-lang/cargo/issues/6139